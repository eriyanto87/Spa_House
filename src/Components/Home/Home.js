import React from "react";

export default function Home() {
  return (
    <div className="home">
      <section>
        <h3>About Evi</h3>
        <p>
          I am a Licensed Massage Therapist, Licensed Esthetician and Licensed
          Nail Technican trained and certified in the State of California. I was
          also trained and certified by Suavislash in Hong Kong for Eyelash
          Extensions. My services include Massage, Skin Care, Nail Care and
          Eyelash Extensions. I am currently located in South Pasadena, CA
          providing services at the comfort of your own home or at my home
          studio for a reduced price.
        </p>
      </section>
      <hr />
      <section>
        <h3>Spa Services At The Comfort Of Your Own Home</h3>
        <p>
          To provide a space of tranquility, wellness and beauty to carry you
          away from the stress of your daily life.{" "}
        </p>
      </section>
    </div>
  );
}
