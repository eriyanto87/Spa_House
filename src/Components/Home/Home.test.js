import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import Home from "./Home";

describe("Home component", () => {
  it("Home Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Home />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
