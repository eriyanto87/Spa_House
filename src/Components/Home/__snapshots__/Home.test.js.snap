// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Home Snapshot 1`] = `
<div
  className="home"
>
  <header>
    <div
      className="nav"
    >
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
        </li>
        <li>
          <a
            href="/about"
            onClick={[Function]}
          >
            About 
          </a>
        </li>
        <li>
          <a
            href="/treatments"
            onClick={[Function]}
          >
            Treatments 
          </a>
        </li>
        <li>
          <a
            href="/pricing"
            onClick={[Function]}
          >
            Pricing 
          </a>
        </li>
        <li>
          <a
            href="/contact"
            onClick={[Function]}
          >
            Contact Us 
          </a>
        </li>
        <li>
          <a
            href="/gallery"
            onClick={[Function]}
          >
            Gallery
          </a>
        </li>
      </ul>
    </div>
  </header>
  <div
    className="bg-text"
  >
    SPA HOUSE
  </div>
  <footer>
    <div>
      <p>
        Â© 2020
      </p>
    </div>
  </footer>
</div>
`;
